assemble.go

```diff
@@ -227,9 +230,16 @@ func AssembleFile(ctx context.Context, name string, idx Index, s Store, seeds []
 			// This plan has at least one invalid seed
 			if options.InvalidSeedAction == InvalidSeedActionBailOut {
 				return stats, err
+			} else if options.InvalidSeedAction == InvalidSeedActionRegenerate {
```

Comment:
Almost time for a `switch options.InvalidSeedAction {` here

Reply:
Done, with a default that panics to ensure that we are actually considering all the possibilities.